external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -962,7 +978,9 @@ fn run(
                 )
             }
             PassResult::Typing(tprog) => {
-                compilation_env.check_diags_at_or_above_severity(Severity::BlockingError)?;
+                compilation_env
+                    .check_diags_at_or_above_severity(Severity::BlockingError)
+                    .map_err(|diags| (PASS_TYPING, diags))?;
```

Comment:
Not necessary, but a thought: could we write a method on PassResult to provide this?

```suggestion
                    .map_err(|diags| cur.tagged_diagnostics(diags))?;
```